diff --git a/scopes/scope/network/http/http.ts b/scopes/scope/network/http/http.ts
index ef2b8d3b9..47480ad89 100644
--- a/scopes/scope/network/http/http.ts
+++ b/scopes/scope/network/http/http.ts
@@ -158,6 +158,7 @@ export class Http implements Network {
   }
 
   static async getNetworkConfig(): Promise<NetworkConfig> {
+    // console.trace('here')
     const obj = listConfig();
 
     const getAsNumber = (key: string): number | undefined => {
diff --git a/scopes/scope/scope/scope-component-loader.ts b/scopes/scope/scope/scope-component-loader.ts
index 43cdb1b85..f61c515f4 100644
--- a/scopes/scope/scope/scope-component-loader.ts
+++ b/scopes/scope/scope/scope-component-loader.ts
@@ -23,6 +23,8 @@ export class ScopeComponentLoader {
   }
 
   async get(id: ComponentID, importIfMissing = true, useCache = true): Promise<Component | undefined> {
+    console.log('scope.get', id.toString(), 'import if missing', importIfMissing);
+    // console.trace(`ScopeComponentLoader.get, id: ${id.toString()}, importIfMissing: ${importIfMissing}, useCache: ${useCache}`);
     const fromCache = this.getFromCache(id);
     if (fromCache && useCache) {
       return fromCache;
@@ -38,6 +40,7 @@ export class ScopeComponentLoader {
       this.scope.isExported(id) &&
       !this.importedComponentsCache.get(id.toString())
     ) {
+      console.log('IMPORTING');
       await this.scope.import([id], { reason: `${id.toString()} because it's missing from the local scope` });
       this.importedComponentsCache.set(id.toString(), true);
       modelComponent = await this.scope.legacyScope.getModelComponentIfExist(id);
diff --git a/scopes/workspace/workspace/workspace.ts b/scopes/workspace/workspace/workspace.ts
index 20c803bc8..7f5ccf6c3 100644
--- a/scopes/workspace/workspace/workspace.ts
+++ b/scopes/workspace/workspace/workspace.ts
@@ -2221,6 +2221,7 @@ the following envs are used in this workspace: ${uniq(availableEnvs).join(', ')}
         loadExtensions: true,
       });
     } catch (error) {
+      console.log("🚀 ~ Workspace ~ resolveEnvManifest ~ error:", error)
       // If component not found in workspace (e.g. during bit new with extends), try remote
       if (error instanceof VersionNotFoundOnFS || (error as any).name === 'VersionNotFoundOnFS') {
         envComponent = await this.scope.getRemoteComponent(resolvedEnvComponentId);
